doc/developers-guide/SCRAM-serialization.md

```diff
@@ -22,19 +22,46 @@ In order to learn more about the meaning of the Stored Key, Server Key, Salt and
 * *Erlang map:*
 ```
 #{iteration_count => 4096,
-   salt => <<"glF/hUXiCWD6TaJLpgVaaw==">>,
-   sha =>
-      #{server_key => <<"ztvci4eBktO3NLviLhwN3ktF15Q=">>,
-        stored_key => <<"Hxfegcj54A6p/ZzWkw7BjJsLZpM=">>},
-   sha256 =>
-      #{server_key => <<"0Tm46vWQyO/XWpAtcDCUH++7ImWVDe0VDx274zMzMXQ=">>,
-        stored_key => <<"S4XiSkYI81m688nMuCgFGUBIGbW6oG1a9rlTKktzS/A=">>}}
+  salt => <<"aml22qUoKvwJHccCCH00eQ==">>,
+  sha =>
+      #{server_key => <<"gkoblYUZnW8GRBhIyJnbflHlLYs=">>,
+        stored_key => <<"+nm4+0ONdpgnoypippxdzV5sQ80=">>},
+  sha224 =>
+      #{server_key =>
+            <<"JwSXbKqMRJEwOr/iNmqN+x3UzxRikmKym9E71g==">>,
+        stored_key =>
+            <<"nUi8YwRBRMAusH/KpINo3/AO32UWzlSONX9wMA==">>},
+  sha256 =>
+      #{server_key =>
+            <<"y0cB/hZ7AKtVMC2WCkXlo4XTNfOQVg30PLfIhK+Wf/U=">>,
+        stored_key =>
+            <<"tiDUGNpvmt75PGcCvwoTLVOF/og/BiX1FOpihXlYqW8=">>},
+  sha384 =>
+      #{server_key =>
+            <<"LJKetdkPytdOXg6aj4NN25KmJatJsl5zaU78bzjowYrBcjG+wux/I5q7E78sQVSn">>,
+        stored_key =>
+            <<"s7SdIo5a+LH/EsKIMoqa4PPEveScCnDwP1LeaAzVdANT5pPSMio/CoMDN4uXfnHr">>},
+  sha512 =>
+      #{server_key =>
+            <<"TCuJFv5dmpshThJbQnURW0LOz7D55d5hgYndA3jdklQd2omL6PpfgfIgToyVvYlsF9sRGYOg255y+Q+ltwW3tQ==">>,
+        stored_key =>
+            <<"4ATRLxRB+d6YyZXxi3PorT6kyS4Mr6tEuKUVhInJcRU0NDpXh94Y/Yrd+EDOSZUnPno8aAzj78NUXOTyoB98rg==">>}}
+```
+* *Serialized password:*
+```
+
+==MULTI_SCRAM==,aml22qUoKvwJHccCCH00eQ==,4096,
+===SHA1===+nm4+0ONdpgnoypippxdzV5sQ80=|gkoblYUZnW8GRBhIyJnbflHlLYs=,
+==SHA224==nUi8YwRBRMAusH/KpINo3/AO32UWzlSONX9wMA==|JwSXbKqMRJEwOr/iNmqN+x3UzxRikmKym9E71g==,
+==SHA256==tiDUGNpvmt75PGcCvwoTLVOF/og/BiX1FOpihXlYqW8=|y0cB/hZ7AKtVMC2WCkXlo4XTNfOQVg30PLfIhK+Wf/U=,
+==SHA384==s7SdIo5a+LH/EsKIMoqa4PPEveScCnDwP1LeaAzVdANT5pPSMio/CoMDN4uXfnHr|LJKetdkPytdOXg6aj4NN25KmJatJsl5zaU78bzjowYrBcjG+wux/I5q7E78sQVSn,
+==SHA512==4ATRLxRB+d6YyZXxi3PorT6kyS4Mr6tEuKUVhInJcRU0NDpXh94Y/Yrd+EDOSZUnPno8aAzj78NUXOTyoB98rg==|TCuJFv5dmpshThJbQnURW0LOz7D55d5hgYndA3jdklQd2omL6PpfgfIgToyVvYlsF9sRGYOg255y+Q+ltwW3tQ==
+
 ```
-* *Serialized password:* `==MULTI_SCRAM==,glF/hUXiCWD6TaJLpgVaaw==,4096,===SHA1===Hxfegcj54A6p/ZzWkw7BjJsLZpM=|ztvci4eBktO3NLviLhwN3ktF15Q=,==SHA256==S4XiSkYI81m688nMuCgFGUBIGbW6oG1a9rlTKktzS/A=|0Tm46vWQyO/XWpAtcDCUH++7ImWVDe0VDx274zMzMXQ=`
 
 ## Legacy format description
 
-MongooseIM installations prior to 3.6.3 were supporting only SHA-1 as a hashing algorithm for SCRAM. The SCRAM format that was used can be seen below.
+MongooseIM installations prior to 3.7 were supporting only SHA-1 as a hashing algorithm for SCRAM. The SCRAM format that was used can be seen below.
```

Comment:
There is no `3.7` yet. It'd probably be better to rephrase the sentence with an existing version like:
```
installations older or equal to 3.6.2
```

Reply:
Sure, I just was thinking that I'm already writing for the future but what you suggest is more accurate and correct.